.github/workflows/maven.yml

```diff
@@ -47,3 +47,7 @@ jobs:
 
       - name: 'Build project'
         run: mvn -B clean verify -Pci
+
+      - name: 'Test submission'
+        run: ./test.sh ${{ github.event.pull_request.user.login }}
```

Comment:
I am not sure this will work for all submissions, e.g. some use sdk or graal

Reply:
Indeed it might not work. I'll see if we can use https://github.com/sdkman/sdkman-action/tree/main

Reply:
Yeah, I think this would be good. Wanna add it to this PR?

Reply:
Was already working on it, but I am travelling today so a bit slow :)

Reply:
There is a related ticket https://github.com/gunnarmorling/1brc/issues/150